lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/component/ComponentPresenter.java

```diff
@@ -35,4 +41,16 @@ public void onConfigurationChanged(ComponentLayout view, Options options) {
         Options withDefault = options.withDefaultOptions(defaultOptions);
         setBackgroundColor(view, withDefault);
     }
+
+    @Nullable
+    public Animator getStatusBarPushAnimation(@NonNull Options appearingOptions) {
+        Options appearingOptionsWithDefault = appearingOptions.copy().withDefaultOptions(defaultOptions);
+        return StatusBarPresenter.instance.getStatusBarPushAnimation(appearingOptionsWithDefault);
+    }
+
+    @Nullable
+    public Animator getStatusBarPopAnimation(@NonNull Options appearingOptions, @NonNull Options disappearingOptions) {
+        Options appearingOptionsWithDefault = appearingOptions.copy().withDefaultOptions(defaultOptions);
+        return StatusBarPresenter.instance.getStatusBarPopAnimation(appearingOptionsWithDefault, disappearingOptions);
+    }
```

Comment:
Is ComponentPresenter the right place to set the status-bar? Need to rethink this